scipy/ndimage/filters.py

```diff
@@ -1,3 +1,8 @@
+# This file is not meant for public use and will be removed in the future
+# versions of SciPy. Use the `scipy.ndimage` namespace for importing the
```

Comment:
If we know what version we plan to remove this, I would mention the specific version here (and in the other files)

Reply:
If we are not listing a specific version, I would just make a slight edit from "in the future versions" to "in a future version"

Reply:
@rgommers mentioned the expiry with SciPy 2.0 in the [message](https://mail.python.org/pipermail/scipy-dev/2021-July/024943.html) to the mailing list.